contents/tutorials/explore-insights-session-recordings.md

```diff
@@ -0,0 +1,76 @@
+---
+title: How to further explore insights with session recordings
```

Comment:
Is there a more clickable headline we could use here?

I'm asking the question because no amazing ideas spring to mind!

Reply:
I tried "How to use session recordings to get a deeper understanding of user behavior"